src/components/stepforms/schemas/join.ts

```diff
@@ -18,7 +18,7 @@ export default {
     },
     type: {
       type: 'string',
-      enum: ['left', 'inner'],
+      enum: ['left', 'inner', 'left outer'],
```

Comment:
Thanks for the fix!

I'm just wondering: shouldn't the `join.ts` schema file be imported in `JoinStepForm.vue` in order to make sure they're always sync?

Reply:
Good suggestion, thanks.